Sources/NostrSDK/Tag.swift

```diff
@@ -15,7 +15,10 @@ public enum TagName: String {
     
     /// points to the id of an event this event is quoting, replying to or referring to somehow
     case event = "e"
-    
+
+    /// coordinates to a replaceable event, which includes the kind number, pubkey that signed the event, and optionally the identifier (if the replaceable event is parameterized)
+    case replaceableEvent = "a"
```

Comment:
Maybe we could call this `eventCoordinates`? The NIPs repo's README says "coordinates to an event" as the value for "a" tags.

Reply:
That's actually what I had initially and changed to `replaceableEvent`. Not sure if I like the naming in the NIPs repo, but going back to our core principles, we should just follow the naming from the spec. I'll change it back to `eventCoordinates`.